### PR TITLE
Fix `base64` Invalid Argument Error on macOS using pre-installed `FreeBSD base64`

### PR DESCRIPTION
**Pull request by nanophate**
# Description

Add additional  `base64` command on macOS to prevent the `invalid argument ~/oidc-service-jwk.json` error by including the `-i` option to specify the input file.

# Reasons

Encountered an error when using the `base64` command on macOS due to different command-line options compared to other Unix-like systems. The provided change ensures users can avoid this error by using the correct options.

```
❯ base64 --version
FreeBSD base64
❯ base64 -b0 ~/oidc-service-jwk.json
base64: invalid argument /Users/vivio/oidc-service-jwk.json
Usage:	base64 [-Ddh] [-b num] [-i in_file] [-o out_file]
  -b, --break    break encoded string into num character lines
  -Dd, --decode   decodes input
  -h, --help     display this message
  -i, --input    input file (default: "-" for stdin)
  -o, --output   output file (default: "-" for stdout)
❯ base64 -b0 -i ~/oidc-service-jwk.json
**[REDACTED]**
```

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [x] Break up walls of text by adding paragraph breaks.
- [x] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [x] Keep the title between 20 and 70 characters.
- [x] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [x] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [ ] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [x] Include relevant backlinks to other CircleCI docs/pages.

### CODE REVIEW

**Comment by rosieyohannan** [comment]
Thank you @nanophate! I just tagged on-prem for review, and we will need to ensure this gets into 4.6 docs branch

File: jekyll/_cci2/server/v4.4/installation/phase-2-core-services.adoc [hunk 1]
```diff
@@ -890,6 +890,13 @@ $ base64 -w0 ~/oidc-service-jwk.json
 $ base64 -b0 ~/oidc-service-jwk.json
 ----
 +
+If you encounter an `invalid argument ~/oidc-service-jwk.json` error on macOS, use this command instead:
++
+[source,shell]
+----
+$ base64 -b0 -i ~/oidc-service-jwk.json
+----
++
```
Comment (christian-stephen) [inline jekyll/_cci2/server/v4.4/installation/phase-2-core-services.adoc:899]: This should just replace the macOS instruction altogether.

Comment (christian-stephen) [reply jekyll/_cci2/server/v4.4/installation/phase-2-core-services.adoc:899]: (for context, I use Linux for development, not macOS, so I had just looked up the manpages for BSD `base64` when I wrote this docs and evidently made an error, let's just replace the invalid command)

Comment (nanophate) [reply jekyll/_cci2/server/v4.4/installation/phase-2-core-services.adoc]: @christian-stephen Thanks for taking a look at this! Just replaced the command to include `-i` option for macOS command. 🙇

**Review by christian-stephen — APPROVED** [review]
Thank you!

**Review by rosieyohannan — APPROVED** [review]

**Pull request merged**